webrtc.html

```diff
@@ -4306,6 +4307,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <h2>Methods</h2>
             <dl data-link-for="RTCCertificate" data-dfn-for="RTCCertificate"
             class="methods">
+              <dt><dfn><code>getAlgorithms</code></dfn></dt>
+              <dd>
+                <p>Returns a sequence of supported certificate algorithms.</p>
+              </dd>
```

Comment:
If a implementation returns an empty sequence, is that compliant with this?

Reply:
Added a requirement that at least one supported algorithm be returned.